GDJS/Runtime/runtimeobject.js

```diff
@@ -4,6 +4,14 @@
  * This project is released under the MIT License.
  */
 
+/**
+ * @typedef {Object} ObjectData Base parameters for all gdjs.RuntimeObject
+ * @property {string} name The name of the object (for getting from RuntimeScene.getObjects for example)
```

Comment:
```
The name of the object. During the game, objects can be queried by their name (see `RuntimeScene.getObjects` for example).
```

Reply:
`The name of the object. During the game, objects can be queried by their name (see {@link gdjs.RuntimeScene.prototype.getObjects} for example).` ?

Reply:
Yes if they work this is ideal.

Reply:
Done